src/pages/en/tutorial/0-introduction/index.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: ~/layouts/TutorialLayout.astro
+unitTitle: Prepare your working environment
```

Comment:
```suggestion
unitTitle: Get started
```
@Jutanium Do you think this is a more appropriate/accurate title for 0.0 Welcome and 0.1 setting up the accounts? Open to ideas here!